protocol/avdl/chat1/local.avdl

```diff
@@ -86,46 +84,46 @@ protocol chatLocal {
   }
 
   record Message {
```

Comment:
Since everything is in `chat1` now, do we want to add "Local" suffix to all types that is only used by the chatLocal service? Same thing for remote only types. Then those types shared/used by both local service and remote service can go to `common.avdl`. I think we could use `gorename` to update the names first, then change avdl and recompile, to avoid changing these by hand. What do you think? 

Although if it's too complicated to change all the names, I'm OK with putting it off as well!